### PR TITLE
Add repair step and revert "Open updater" button

### PR DESCRIPTION
**Pull request by MorrisJobke**
- fixes both items in #1485
- repair step moves the update step file `data/updater-INSTANCEID/.step` to a file named `.step-previous-update` to not show all steps as done on the next update
- reverts  202ae42

cc @LukasReschke @nickvergessen @karlitschek 

I will open the PRs for stable9 and stable10 soon

### CODE REVIEW

**Comment by karlitschek** [comment]
nice 👍  As discussed!

**Comment by MorrisJobke** [comment]
needed to rebase to fix the sign-off message in the commits ;)

**Comment by nickvergessen** [comment]
Looks good otherwise

File: apps/updatenotification/tests/UpdateCheckerTest.php [hunk 1]
```diff
@@ -65,16 +64,14 @@ public function testGetUpdateStateWithUpdateAndValidLink() {
...
+				'web'=> 'https://owncloud.org/myUrl',
 			]);
 
 		$expected = [
 			'updateAvailable' => true,
-			'updateVersion' => 'Nextcloud 123',
-			'updateLink' => 'https://docs.nextcloud.com/myUrl',
-			'downloadLink' => 'https://downloads.nextcloud.org/server',
+			'updateVersion' => 'ownCloud 123',
+			'updateLink' => 'https://owncloud.org/myUrl',
```
Comment (MorrisJobke) [inline apps/updatenotification/tests/UpdateCheckerTest.php]: True story. I will fix it.

**Comment by MorrisJobke** [comment]
I added the wrongly removed tests here.

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**